src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/PreferredPerson/PreferredPersonCellModel.swift

```diff
@@ -4,8 +4,9 @@
 
 import Foundation
```

Comment:
```suggestion

```
maybe remove Foundation as it is already included in UIKit

Reply:
Ah, since this is a model I removed the UIKit import instead, don't know how that got in there 😇

Reply:
oh ok , then all is good